app/Nova/Resources/UiTiDv1.php

```diff
@@ -35,18 +36,19 @@ final class UiTiDv1 extends Resource
 
     public static $displayInNavigation = false;
 
-    /**
-     * @var array<string>
-     */
-    public static $search = [
-        'id',
-        'integration_id',
-        'consumer_id',
-        'consumer_key',
-        'consumer_secret',
-        'api_key',
-        'environment',
-    ];
+    public static $searchable = false;
+
+    public static function defaultOrderings($query): Builder
+    {
+        /** @var Builder $query */
+        return $query->orderByRaw(
+            'CASE
+                WHEN environment = \'acc\' THEN 1
+                WHEN environment = \'test\' THEN 2
+                WHEN environment = \'prod\' THEN 3
+            END'
```

Comment:
Since you have the same code in 2 places.
Maybe you could write a reusable function `orderEnvironments?`